lib/ASN1/Base128.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace FG\ASN1;
+
+/**
+ * A base-128 decoder.
+ */
+class Base128
+{
+    /**
+     * @param integer $value
+     *
+     * @return string
+     */
+    public static function encode($value)
+    {
+        $octets = chr($value & 0x7F);
+        $value >>= 7;
+
+        while ($value > 0) {
+            $octets .= chr(0x80 | ($value & 0x7F));
+            $value >>= 7;
+        }
+
+        return strrev($octets);
```

Comment:
I compared your implementation with the one from three other DER encoding implementations (openssl, golang and ASN.1 JavaScript decoder) and they result in the same encoding so it seems like I misunderstood the standard about the encoding at this point.

From what I can reconstruct the correct encoding of `0x80`is:

```
value:   0x80
binary:  10000000
septets:  0000001  0000000
padded:  10000001 00000000
hex:     0x81     0x00
```

It actually does make more sense this way so feel free to ignore this commet block.
Sometimes it just helps writting down those things to get a better understanding of the whole thing :)